.install/linux_install_cmake.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+wget https://github.com/Kitware/CMake/releases/download/v3.24.3/cmake-3.24.3-linux-x86_64.sh 
```

Comment:
Can you remove the version - up a level maybe? Then all your versions come from one place (fewer things to maintain)